### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by kevinch**
Fake PR for Bot testing purposes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,35 +12,30 @@ npm install # or yarn!
 npm run dev
 ```
 
-Open up [localhost:3000](http://localhost:3000) and start clicking around.
-
-Consult [sapper.svelte.dev](https://sapper.svelte.dev) for help getting started.
-
+- Open up [localhost:3000](http://localhost:3000) and start clicking around.
+- Consult [sapper.svelte.dev](https://sapper.svelte.dev) for help getting started.
```
Comment (kaisermann) [inline README.md]: absurdo

File: README.md [hunk 2]
```diff
@@ -12,35 +12,30 @@ npm install # or yarn!
...
 
 ### src
 
 The [src](src) directory contains the entry points for your app — `client.js`, `server.js` and (optionally) a `service-worker.js` — along with a `template.html` file and a `routes` directory.
 
-
 #### src/routes
 
-This is the heart of your Sapper app. There are two kinds of routes — *pages*, and *server routes*.
+This is the heart of your Sapper app. There are two kinds of routes — _pages_, and _server routes_.
```
Comment (kaisermann) [inline README.md]: noooooo

File: README.md (hunk 1 above)
Comment (kevinch) [reply README.md]: 😭

File: README.md [hunk 3]
```diff
@@ -12,35 +12,30 @@ npm install # or yarn!
...
-* The file `src/routes/index.svelte` (or `src/routes/index.js`) corresponds to the root of your app. `src/routes/about/index.svelte` is treated the same as `src/routes/about.svelte`.
-* Files and directories with a leading underscore do *not* create routes. This allows you to colocate helper modules and components with the routes that depend on them — for example you could have a file called `src/routes/_helpers/datetime.js` and it would *not* create a `/_helpers/datetime` route
-
+- A file called `src/routes/about.svelte` corresponds to the `/about` route. A file called `src/routes/blog/[slug].svelte` corresponds to the `/blog/:slug` route, in which case `params.slug` is available to the route
+- The file `src/routes/index.svelte` (or `src/routes/index.js`) corresponds to the root of your app. `src/routes/about/index.svelte` is treated the same as `src/routes/about.svelte`.
+- Files and directories with a leading underscore do _not_ create routes. This allows you to colocate helper modules and components with the routes that depend on them — for example you could have a file called `src/routes/_helpers/datetime.js` and it would _not_ create a `/_helpers/datetime` route
```
Comment (kaisermann) [inline README.md]: oh god

**Pull request closed without merging**